### ISSUE TITLE
Unable to display hidden tabs 

### ISSUE THREAD

**Issue by browetd** [maykar/custom-header#121]
From my main tab, I click on a button (I am using button-card add-on) with a path to a hidden tab, After a very quick display of the tab (less than 1 second), It comes back immediately to the main tab (phone and PC, same behaviour)... If I choose to display permanently the "hidden" tab, it stays on it... I had that behaviour in a previous version of Custom Header but it was fixed at that time... More information: If I completely disable Custom header, all the tabs are visible on the top of my screen but the problem remains the same with previously "hidden" tabs, If I choose one of the tab by clicking on it on the top of the screen, it goes back to the main tab...  So to be displayed and stay on it, it has to be in the "show_tabs" list.

My config: Home Assistant 0.104.3 in a venv on a raspberry Pi4 and Python 3.7.3.

Custom Header has been installed with HACS... If I go back to a previous version of Custom Header the problem stays ... while I am sure it was working in previous versions, I went back to 1.2.7 without success, the problem remains... Strange...
My custom header (version 1.3.2) config (I have 15 tabs in total with only two displayed):

`  - url: /community_plugin/custom-header/custom-header.js
    type: module

custom_header:
  compact_mode: "true"
  header_text: 'Maisons Connectées'
  show_tabs:  "general, tabs"
`

**Comment by maykar** [comment]
This was actually a fixed bug, you shouldn’t have been able to navigate to a hidden tab with hidden_tab_redirect on (which is the default). There were a few ways around it, including using navigation links in other cards (as you’ve found out), but now the redirect works as intended.

TLDR: You just need to add `hidden_tab_redirect: false`

**Comment by browetd** [comment]
I missed that parameter... It is working now !!!!  Issue closed.  Thanks
